rust-backend/Dockerfile

```
@@ -0,0 +1,68 @@
+
+# Create app directory
+WORKDIR /app
+
+# Copy the binary from builder stage
+COPY --from=builder /app/target/release/weather-server /app/weather-server
+
+# Copy templates
+COPY --from=builder /app/templates ./templates
+
+# Copy static files (if they exist)
+COPY --from=builder /app/static ./static 2>/dev/null || true
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)